composer.json

```diff
@@ -25,7 +25,9 @@
     "phpunit/phpunit": "^7.5",
     "brain/monkey": "^2.3",
     "giacocorsiglia/wordpress-stubs": "^5.1",
```

Comment:
@szepeviktor I guess, this depencency should be removed as well?

Reply:
Oh yesss!

Reply:
Done.